### PR TITLE
Adding ensureTodoConfig to write default config

### PR DESCRIPTION
**Pull request by scalvert**
Adds new `ensureTodoConfig` directory.

```ts
export function ensureTodoConfig(baseDir: string): void;
```

This allows us to provide this behavior in the respective consuming packages, `@scalvert/eslint-formatter-todo` and `ember-template-lint`.

### CODE REVIEW

File: __tests__/todo-config-test.ts [hunk 1]
```diff
@@ -160,6 +161,62 @@ describe('todo-config', () => {
...
+      });
+
+      const originalPkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });
+
+      ensureTodoConfig(project.baseDir);
+
+      const pkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });
+
+      expect(pkg).toEqual(originalPkg);
+    });
```
Comment (rwjblue) [inline __tests__/todo-config-test.ts:192]: Can you add another test that confirms that it doesn't add config if an empty config is found (either as `{}` or `null`)?

Something like:

```js
    it('does not change package.json if todo config is empty object', () => {
      ensureTodoStorageDirSync(project.baseDir);

      writeTodoConfig(project.baseDir, {});

      const originalPkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });

      ensureTodoConfig(project.baseDir);

      const pkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });

      expect(pkg).toEqual(originalPkg);
    });

    it('does not change package.json if todo config is null', () => {
      ensureTodoStorageDirSync(project.baseDir);

      writeTodoConfig(project.baseDir, null);

      const originalPkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });

      ensureTodoConfig(project.baseDir);

      const pkg = readFileSync(join(project.baseDir, 'package.json'), { encoding: 'utf8' });

      expect(pkg).toEqual(originalPkg);
    });
```

File: __tests__/todo-config-test.ts [hunk 2]
```diff
@@ -160,6 +161,62 @@ describe('todo-config', () => {
...
+
+    it('does changes package.json if lintTodo directory not present', () => {
```
Comment (rwjblue) [inline __tests__/todo-config-test.ts:194]:
```suggestion
    it('changes package.json if lintTodo directory not present', () => {
```

**Review by rwjblue — APPROVED** [review]
Looks great! Only one small suggestion about a couple of additional tests, but implementation looks good to me.

File: __tests__/todo-config-test.ts (hunk 1 above)
Comment (scalvert) [reply __tests__/todo-config-test.ts:192]: Well I think I'll just steal yours :)

Comment (scalvert) [reply __tests__/todo-config-test.ts:192]: So the second test case won't work unless I change the signature of the `writeTodoConfig` function to accept `null`. I'll just add the single case.

File: __tests__/todo-config-test.ts (hunk 2 above)
Comment (scalvert) [reply __tests__/todo-config-test.ts:208]: But it does changes :)

Comment (scalvert) [reply __tests__/todo-config-test.ts:208]: Fixed.
